C++/C++.sublime-syntax

```diff
@@ -1129,6 +1129,23 @@ contexts:
     - match: (?=\W)
       pop: true
 
+  global-maybe-constant-or-function:
+    - include: comments
+    - include: types
+    - include: modifiers
+    - include: constants
+    - include: variables
+    - include: scope:source.c#named-constants
```

Comment:
The included `constants` context of line 1136 already includes `scope:source.c#comments` which itself includes `named-constants`. So this include seems duplicated and should be removed.

Doing so would also make this context obsolete as the pattern could be moved back into _constants_.